packages/react-interactive-tab/stories/InteractiveTab/Default.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
```

Comment:
This file should be removed before merge, as we are going to import these hooks from the @fluentui/react-tabs once the https://github.com/microsoft/fluentui/pull/32125 merged and the package is published.